packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.controller.js

```diff
@@ -62,40 +92,20 @@ export default class PciTrainingJobsSubmitController {
       });
   }
 
-  onAddVolume(form) {
-    const volume = {
-      region: form.container.$viewValue.region,
-      container: form.container.$viewValue.name,
-      mountPath: form.mountPath.$viewValue,
-      permission: form.permission.$viewValue,
-    };
-
-    this.job.volumes.push(volume);
-    this.filterContainers();
-  }
-
-  // Dirty hack to validate the oui-inline-adder form
-  // eslint-disable-next-line class-methods-use-this
-  forceSubmitContainer(form) {
-    form.$$controls.forEach((innerForm) => {
-      // eslint-disable-next-line no-param-reassign
-      innerForm.$error = {};
-      innerForm.$setPristine();
+  validateVolume() {
+    this.job.validVolume = true;
+    map(this.httpHeader, (volume) => {
+      if (!volume.added && has(volume, 'model.container')) {
```

Comment:
Could be replaced by a native `map` as it's a variable declared in the `$onInit` and the `?` operator to avoid lodash imports

Reply:
Please check if its okay now